File: android/app/src/main/java/com/uchannel/model/Message.kt
package com.uchannel.model

import java.util.Date

/**
 * 消息数据类
 */
data class Message(
    val id: String,
    val content: String,
    val sender: MessageSender,
    val timestamp: Date = Date()
)

enum class MessageSender {
    USER,      // 用户消息
    ASSISTANT  // 助手消息
}

File: react-native-app/src/models/Message.ts
export enum MessageSender {
  USER = 'USER',
  ASSISTANT = 'ASSISTANT',
  SUGGESTION = 'SUGGESTION',
}

export interface Message {
  id: string;
  content: string;
  contentSecondary?: string;
  sender: MessageSender;
  timestamp: number;
  scheduleTitle?: string;
  scheduleTime?: string;
  scheduleLocation?: string;
}
